linear_damping_eigenvalues:       [80.0, 80.0, 80.0]

angular_damping_eigenvalues:      [5., 5., 5.]

#q_nullgains:                      [0.5,0.5,0.5,0.5,0.5,0.5,1.]
q_nullgains:                      [1,60,10,40,5,1,1]
#q_nullpose:                       [0.044752691045324394, 0.6951627023357917, -0.01416978801753847, -1.0922311725109015, -0.0050429618456282, 1.1717338014778385, -0.01502630060305613]
#q_nullpose:                       [ 0.        ,  0.84823002,  0.        , -1.12713363,  0.        ,-0.36477381,  0.        ]
                                  #deg [0.0, 48.6, 0.0, -64.58, 0.0, -20.9, 0.0]
q_nullpose:                       [ 0.        ,  0.52359878,  0.        , -1.04719755,  0.  ,1.57079633 ,  0.        ]  

external_tool_compensation:       [0, 0, 0.0, 0, 0.0, 0.0]
